### PR TITLE
changed notifyByText name and complete resolver and services for upda…

### PR DESCRIPTION
**Pull request by ramyx**
-- changed "notifyByTxt" by "notifyByText"
-- Fix updateProfile in UserResolver
-- Added service updateUser in UserService

### CODE REVIEW

File: src/graphql/user/user.resolver.ts [hunk 1]
```diff
@@ -81,10 +81,14 @@ export class UserResolver {
...
+    this.logger.log('user', reqUser);
+    const { userKey } = reqUser;
 
-    const updatedUser = await UserEntity.findOne( {userKey} );
-    return updatedUser;
+    const updatedUser = await this.userService.updateUser(userKey, input);
+    if (!updatedUser) {
+      throw new Error('User not exists');
+    }
+    return await UserEntity.findOne(userKey);
```
Comment (brianschardt) [inline src/graphql/user/user.resolver.ts]: why would we find the user by hitting the database when we already have the most up to date instance of the user here?

File: src/shared/services/user.service.ts [hunk 2]
```diff
@@ -29,6 +30,14 @@ export class UserService {
     return user;
   }
 
+  async updateUser(userKey: string, newUser: UserEntity): Promise<UpdateResult> {
```
Comment (brianschardt) [inline src/shared/services/user.service.ts]: Not sure what the purpose of this is. As it is very simple functionality that the orm provides us. Do we need a separate functionality?

Comment (ramyx) [reply src/shared/services/user.service.ts]: Mmm you're right

File: src/graphql/user/user.resolver.ts (hunk 1 above)
Comment (ramyx) [reply src/graphql/user/user.resolver.ts]: I will verify that this instance has the updated data and will correct

**Pull request closed without merging**